utilities.py

```diff
@@ -20,3 +20,19 @@ def paginate_model(model_name, data, filters):
     last = url_for(model_name)+'?page='+str(count/size+1)+'&'+urlencode(filters)
 
     return (next, previous, last, count)
+
+# When multiple filters are passed to a search request, each filter option is encoded
+#   as a separate paramater, e.g. 'rock_types=Gneiss&rock_types=Slate'.  The API
+#   doesn't like this, instead preferring comma-separated values in one string.  This
+#   fixes that.
+def multi_args_to_list(argsIn):
```

Comment:
Naming should be clearer. What are 'args'? Isn't this returning a csv, not a list?

Comments are great, but "well written code needs no comments"